build/source/dshare/popMetadat.f90

```diff
@@ -819,100 +846,105 @@ subroutine popStat(meta,lineWords,nWords,indexFlags,err,message)
  ! initiate error handling
  err=0; message='popStat/'
 
+ ! get the number of words in a line
+ nWords = size(lineWords)
+
  ! make sure the variable only has one output frequency
- found = .false.
- do iFreq = 1,nFreq
-  do iStat = 1,maxVarStat
-   if (meta%statFlag(iStat)) then
-    if (found) then
-     err=20
-     message=trim(message)//'cannot have more than one file line per variable:'//trim(meta%varName)
-     return
-    end if
-    found = .true.
-    exit
-   end if
-  end do
- end do
+ if(count(meta%statFlag)>0)then
+  message=trim(message)//'model output for variable '//trim(meta%varName)//' already defined (can only be defined once)'
+  err=20; return
+ endif
```

Comment:
is it necessary to be a fatal error? What if the user wants to write both monthly and daily runoff.